apps/admin_app/lib/admin_app_web/templates/product/form.html.eex

```diff
@@ -167,16 +167,30 @@
               <%= for option_type <- get_option_types(@parent_product) do %>
               <th><%= option_type.display_name %></th>
               <% end %>
+              <th>State</th>
+              <th>Activate</th>
               <th></th>
             </tr>
           </thead>
           <tbody>
             <%= for variant <- @parent_product.variants do %>
-            <tr>
+            <tr class="product_variants">
               <td><%= variant.id %></td>
               <%= for option <- variant.options do %>
               <td class="option-value" contenteditable="true" data-option_value_id="<%= option.id %>"><%= option.value %></td>
               <% end %>
+              <td class="variant_state"><%= variant.state %></td>
```

Comment:
We are using the content of td tag as state for the update. I think using it from `data attribute` would be proper.

Reply:
Ok so `variant_state` is used as a selector to update the state in HTML. LGTM